src/App.js

```diff
@@ -45,52 +87,47 @@ class Image extends React.Component {
                     return response.json();
                 })
                 .then((arrayOfData) => {
-                    this.setState(prevState => ({
-                        images: prevState.images.concat(arrayOfData),
-                        isFetching: false
-                    }));
+                    this.setState((prevState) => {
+                        const buffArray = prevState.images.concat(arrayOfData)
+                        return {
+                            count: buffArray.length,
+                            images: buffArray,
+                            isFetching: false
+                        }
+                    });
                 });
         }
     }
 
-
     render() {
+        const divStyle = {
+            position: 'sticky',
+            top: '20px',
+            marginLeft: '400px',
+            width: '200px',
+        };
+
+        const handleCountChange = (countToDelete) => {
```

Comment:
It's better to move this function to the `App` class (similar to the `handleScroll` function).

The reason for it is that you are re-creating the function on each render, and then you are passing the **new** instance of this function to the `onCountChange` prop of the `InputForm` component. This makes the `InputForm` component render each time, even if `countToDelete` doesn't change.